### DISCUSSION TITLE
Possible to support S2 Mini

### DISCUSSION THREAD

**Discussion by bsaurusrex** [thorrak/brewpi-esp#68] [Ideas]
Is it possible to support S2 Mini with esp32 branch while still using the custom PCB/sensor board for esp8266?

**Comment by thorrak** [comment]
Do you have a link to the specific board you were thinking of?

**Reply by bsaurusrex** [reply]
https://www.wemos.cc/en/latest/s2/s2_mini.html

**Reply by thorrak** [reply]
Interesting form factor! It's possible, but it would require custom firmware as it's a different pin configuration than the other ESP32 firmware I've been working on.

Out of curiosity, what prompted you to want to use this board?

**Reply by bsaurusrex** [reply]
I would like to go esp32 and get rid of a Tiltbridge.  But I have the sunk cost of a few esp8266 pcb and sensor boards.  

S2 mini seems like the perfect solution.

**Reply by thorrak** [reply]
Ahh, got it.

Unfortunately, @lbussy and I were speaking on this one and he reminded me that the S2 chip is the one that is the single core, no bluetooth version, meaning that it is effectively just an ESP8266 with more GPIO pins. Absent the bluetooth radio it won't be able to run the TiltBridge firmware,  unfortunately.

Even to the extent that it can run the TiltBridge firmware, right now these chips aren't able to run multiple sets of firmware at once meaning that you wouldn't be able to combine a TiltBridge with BrewPi.

**Comment by thorrak** [comment]
Just to close this out - the latest firmware (v15) now supports the ESP32-S2.
